Bump pulseaudio to 16.99.1 release candidate

i-garrison opened the pull request:
- add and unmask `media-libs/webrtc-audio-processing-1.3`
- bump `media-libs/libpulse-16.99.1` with minor fix from master
- bump `media-sound/pulseaudio-daemon-16.99.1` with minor fix from master

leio commented:
Don't we want to stable `media-libs/libpulse-16.1-r3` and `media-sound/pulseaudio-daemon-16.1-r8` earlier than this release candidate, which we might not stable at all? Referring to the `git mv` instead of adding new versions

i-garrison commented:
> Don't we want to stable `media-libs/libpulse-16.1-r3` and `media-sound/pulseaudio-daemon-16.1-r8` earlier than this release candidate, which we might not stable at all? Referring to the `git mv` instead of adding new versions

As I do not have really strong opinion regarding stabling 16.99.1 in a month, I just dropped the 'git mv' part. Resulting change is a bit harder to see in the diff output but guess that's fine.

I also added `meson-multilib` to `webrtc-audio-processing` since that will be needed in future for `gst-plugins-webrtc` anyway.
